components/uuf-renderablecreator-hbs/src/main/java/org/wso2/carbon/uuf/renderablecreator/hbs/helpers/registry/InitHelperRegistry.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.uuf.renderablecreator.hbs.helpers.registry;
+
+import com.github.jknack.handlebars.HelperRegistry;
+import org.wso2.carbon.uuf.renderablecreator.hbs.helpers.init.LayoutHelper;
+import org.wso2.carbon.uuf.renderablecreator.hbs.helpers.init.SecuredHelper;
+
+/**
+ * Default initial implementation of {@link HelperRegistry} used in UUF.
+ * Secured, Layout, I18n and Missing helper is used in this registry
+ */
+public class InitHelperRegistry extends RuntimeHelperRegistry {
+
+    /**
+     * Register built-in and default helpers. We are not registering some of the unwanted helpers (partial, embedded,
+     * i18n. etc) as they are replaced by the custom helpers written for UUF.
+     *
+     * @param registry The handlebars instance.
+     */
+    protected void registerDefaultHelpers(final HelperRegistry registry) {
+        registry.registerHelper(SecuredHelper.HELPER_NAME, new SecuredHelper());
```

Comment:
Put a comment saying that we only register init (pre-compile time) helpers from `helpers.init` package.